webapp-django/crashstats/api/static/api/css/documentation.less

```diff
@@ -1,24 +1,36 @@
-p.url {
-    margin: 15px;
-    font-size: 1.6em;
-}
-p.url code {
-    padding-left: 25px;
+@import "@{root-path}/base/variables.less";
+
+p {
+    &.url {
```

Comment:
This could also just be `p.url` since it's the only thing in the block. Is this an explicit style choice?

I'm not against it or anything, I just noticed it and was curious.

Reply:
I think I just converted the code in the file from top-to-bottom and simply didn't notice that this selector didn't contain much. I've collapsed it down to `p.url`.